packages/syntax-core/src/TapArea/TapArea.tsx

```diff
@@ -51,13 +50,11 @@ type TapAreaProps = AriaAttributes & {
    * * `none`: 0px
    * * `sm`: 8px
    * * `md`: 12px
-   * * `lg`: 16px
-   * * `xl`: 32px
    * * `full`: 999px
    *
    * @defaultValue "none"
    */
-  rounding?: "xl" | "lg" | "md" | "sm" | "full" | "none";
+  rounding?: "md" | "sm" | "full" | "none";
```

Comment:
Seeing places in cambly-frontend where `rounding` is "lg" (https://github.com/Cambly/Cambly-Frontend/blob/main/src/app/components/CamblyNews/ArticlePreviewCard.tsx#L37)
I assume we're just going to update those along with the upgrade right after merging this?

Reply:
@zprong yes, see https://github.com/Cambly/Cambly-Frontend/pull/9298 - seems like that file also updated https://github.com/Cambly/Cambly-Frontend/pull/9298/files#diff-20684dae3c06753b32f2a4de9fb0c3751ffde7e9f234d272d146c3fa32d5a8b7